packages/main/test/specs/Input.spec.js

```diff
@@ -513,42 +508,33 @@ describe("Input general interaction", () => {
 		assert.strictEqual(await valueNotSelected, true, "Value is no longer type aheaded (autocompleted)");
 	});
 
-	it("should remove input's focus when group header item is clicked", async () => {
+	// unstable in headless mode
+	it.skip("should remove input's focus when group header item is clicked", async () => {
 		await browser.url(`test/pages/Input.html`);
 
-		const input = await browser.$("#myInputGrouping");
-		const respPopover = await input.shadow$("ui5-responsive-popover");
-		const groupHeader = await respPopover.$("ui5-list").$("ui5-li-group-header");
+		const input = await $("#myInputGrouping");
 
 		await input.click();
 		await input.keys("C");
+
+		const groupHeader = await input.$("ui5-suggestion-item-group").shadow$("[ui5-li-group-header]");
+
 		await groupHeader.click();
 
 		assert.strictEqual(await groupHeader.getProperty("focused"), false, "Group header is not focused");
 		assert.strictEqual(await input.getProperty("focused"), true, "Input is focused");
-
 	});
 
 	it("checks if the suggestions popover width is minimum the size of the input", async () => {
 		const input = await browser.$("#myInputGrouping");
-		const listItem = await input.shadow$("ui5-responsive-popover").$("ui5-li-suggestion-item");
+		const listItem = await input.$("ui5-suggestion-item");
 
 		await input.click();
 		await input.keys("a");
 
 		assert.strictEqual(await input.getSize('width'), await listItem.getSize('width'), "Suggestions' popover width is minimum the size of the input");
 	});
 
-	it("checks if suggestions popover width is maximum 40rem if input isn't wider", async () => {
```

Comment:
is there a reason why this test is deleted

Reply:
changed it to test the sample above